### PR TITLE
Changing log command to show informations about specialized data.

### PR DESCRIPTION
**Pull request by gilMars**
If you have a data specialization plugin installed and properly confiure, the log command will show extra information about the specialized data.

Output example:
```
ml-git dataset log dataset-ex
INFO - Repository: 

Tag: parquet__dataset-ex__1
Author: Author
Email: youremail@example.com
Date: 2020-12-08 14:10:52
Message: parquet/dataset-ex
Files:
        parquet: ROWS: 10 | COLUMNS: 8


Tag: parquet__dataset-ex__2
Author: Author
Email: youremail@example.com
Date: 2020-12-08 16:11:06
Message: parquet/dataset-ex
Files:
        parquet: ROWS: 11 | COLUMNS: 8

        ++++++++++ Insertions ++++++++++
              [ROWS: 1] | [COLUMNS: 0]
```

### CODE REVIEW

**Review by Mun1z — CHANGES_REQUESTED** [review]
The "new_log_info" variable, it is not very clear what it really is, could you put a more intuitive name?

**Comment by gilMars** [comment]
> The "new_log_info" variable, it is not very clear what it really is, could you put a more intuitive name?

Changed in commit 332514d

**Review by Mun1z — APPROVED** [review]

File: docs/plugins.md [hunk 1]
```diff
@@ -24,6 +24,23 @@ def add_metadata(work_space_path, metadata):
 ```
 </details>
 
+<details>
+<summary><code> compare_metadata </code></summary>
+</br>
+
+Responsible for displaying a formatted output containing, the number of rows and columns of the versioned tabular data,  and the amount of insertions or deletions between each version.
```
Comment (lucasncabral) [inline docs/plugins.md:31]: This document should describe the contract for the development of a plugin. This description provided directly targets a plugin that handles tabular data, but may not be the case. Please add a more general description.

Another question that has arisen is whether the display is made directly by the plugin or if it returns the information to ml-git to format and present it in the manner mentioned in the description.

File: ml_git/_metadata.py [hunk 2]
```diff
@@ -340,7 +340,8 @@ def get_current_tag(self):
     def __sort_tag_by_date(self, elem):
         return elem.commit.authored_date
 
-    def get_log_info(self, spec, fullstat=False):
+    def get_log_info(self, spec, new_info=None, fullstat=False):
```
Comment (lucasncabral) [inline ml_git/_metadata.py:343]: Please rename "new_info" variable to a more intuitive name. Also keep the order of the parameters that already existed, adding the new one at the end.

File: ml_git/_metadata.py [hunk 3]
```diff
@@ -350,9 +351,43 @@ def get_log_info(self, spec, fullstat=False):
...
 
         return formatted
 
+    @staticmethod
+    def _get_spec_content_from_ref(ref, spec_path):
+        entity_spec = ref.tree / spec_path
+        return io.BytesIO(entity_spec.data_stream.read())
+
+    @staticmethod
+    def _parse_tag_to_path(tag):
```
Comment (lucasncabral) [inline ml_git/_metadata.py:366]: Apparently, this method doesn't just turn the tag into a path. It is returning the path to an entity's spec file. If so, why not use the `spec.search_spec_file` or `utils.get_path_with_categories` method?

File: docs/plugins.md (hunk 1 above)
Comment (lucasncabral) [reply docs/plugins.md]: Done in 1e416f2

File: ml_git/_metadata.py (hunk 2 above)
Comment (lucasncabral) [reply ml_git/_metadata.py]: Done in 1e416f2

File: ml_git/_metadata.py (hunk 3 above)
Comment (lucasncabral) [reply ml_git/_metadata.py]: Done in 1e416f2

**Review by lucasncabral — APPROVED** [review]

**Review by andersonpablosilva — APPROVED** [review]

**Review by diego-pedroso — APPROVED** [review]

**Pull request merged**